internal/attestors/vsa/vsa.go

```diff
@@ -191,7 +192,14 @@ func evaluatePolicy(ctx context.Context, opts *config.VsaCommandOptions, attesta
 }
 
 func downloadOPABundle(ctx context.Context, opts *config.VsaCommandOptions, outputFilepath string) error {
-	resp, err := http.Get(opts.PolicyUrl.String())
+	client := http.Client{Timeout: time.Minute}
```

Comment:
I switched this to a dedicated client rather than the default, because the default doesn't have a timeout. And with the number of dependencies we're bringing in, it's also possible that something is modifying the default client as well.